Update package-security.asciidoc (#103492)
elastic/elasticsearch: docs/reference/setup/install/package-security.asciidoc

* Certificates and keys for TLS are generated for the transport and HTTP layer,
and TLS is enabled and configured with these keys and certificates.

> The password and certificate and keys are output to your terminal. If that's not the case, reset the password for the user `elastic` with <<reset-password,`elasticsearch-reset-password`>>.

Reviewer: ```suggestion
The password and certificate and keys are output to your terminal.
You can reset the password for the user `elastic` with the <<reset-password,`elasticsearch-reset-password`>> command.
```
